### ISSUE TITLE
VLAN Interface

### ISSUE THREAD

**Issue by 8ear** [OpenMediaVault-Plugin-Developers/openmediavault-autoshutdown#13]
Hi,
your tool is very great, but i have now a interface eth0 without a ip-adress and a interface eth0.3 from my vlan 3 with my ip.

but now your tool does not regonize this ip. and with the force_nic option i can't select it.

can you make a fix? oder is there any workaround?

LOG OUTPUT:
```
Oct 25 11:20:34 omv logger: autoshutdown[32037]: WARN: ' Invalid parameter format: FORCE_NIC'
Oct 25 11:20:34 omv logger: autoshutdown[32037]: WARN: ' You set it to 'eth0.3', which is not a correct syntax. It has to match '[a-z]{3,}[0-9]{1}''
Oct 25 11:20:34 omv logger: autoshutdown[32037]: WARN: ' with spaces between every NIC: e.g. "eth1 wlan0 usb3"'
Oct 25 11:20:34 omv logger: autoshutdown[32037]: WARN: ' Unsetting FORCE_NIC'
Oct 25 11:20:34 omv logger: autoshutdown[32037]: INFO: ' ------------------------------------------------------'
Oct 25 11:20:34 omv logger: autoshutdown[32037]: INFO: ' Reading NICs ,IPs, ...'
Oct 25 11:20:34 omv logger: autoshutdown[32037]: INFO: ' NIC 'bond0' not found, skipping 'bond0''
Oct 25 11:20:34 omv logger: autoshutdown[32037]: INFO: ' NIC 'eth0' found: try to get IP'
Oct 25 11:20:34 omv logger: autoshutdown[32037]: INFO: ' _check_networkconfig(): Run: #1: No internet-Adress found - wait 60 sec for initializing the network'
Oct 25 11:21:34 omv logger: autoshutdown[32037]: INFO: ' _check_networkconfig(): Run: #2: No internet-Adress found - wait 60 sec for initializing the network'
Oct 25 11:22:34 omv logger: autoshutdown[32037]: INFO: ' _check_networkconfig(): Run: #3: No internet-Adress found - wait 60 sec for initializing the network'
Oct 25 11:23:34 omv logger: autoshutdown[32037]: INFO: ' _check_networkconfig(): Run: #4: No internet-Adress found - wait 60 sec for initializing the network'
Oct 25 11:24:34 omv logger: autoshutdown[32037]: INFO: ' _check_networkconfig(): Run: #5: No internet-Adress found - wait 60 sec for initializing the network'
Oct 25 11:25:34 omv logger: autoshutdown[32037]: WARN: ' No internet-Adress for eth0 found after 5 minutes - The script will not work maybe ...'
Oct 25 11:25:34 omv logger: autoshutdown[32037]: INFO: ' 'eth0' has IP: '
Oct 25 11:25:34 omv logger: autoshutdown[32037]: WARN: ' Invalid parameter format: Class: nnn.nnn.nnn]'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' It is set to '', which is not a correct syntax. Maybe parsing 'ifconfig ' did something wrong'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' Please report this Bug and the CLI-output of 'ifconfig''
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' unsetting NIC[2]: eth0 ...'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' Invalid parameter format: SERVERIP [iii]'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' It is set to '', which is not a correct syntax. Maybe parsing 'ifconfig' did something wrong'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' Please report this Bug and the CLI-output of 'ifconfig''
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' unsetting NIC[2]:  ...'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: INFO: ' NIC 'eth1' not found, skipping 'eth1''
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' No SERVERIP or CLASS found'
Oct 25 11:25:35 omv logger: autoshutdown[32037]: WARN: ' exiting ...'
```

ifconfig OUTPUT:
```
eth0      Link encap:Ethernet  Hardware Adresse d0:50:99:3a:0c:89  
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metrik:1
          RX packets:685518 errors:0 dropped:0 overruns:0 frame:0
          TX packets:984367 errors:0 dropped:0 overruns:0 carrier:0

eth0.3    Link encap:Ethernet  Hardware Adresse d0:50:99:3a:0c:89  
          inet Adresse:192.168.160.67  Bcast:192.168.160.127  Maske:255.255.255.192
          UP BROADCAST RUNNING MULTICAST  MTU:1500  Metrik:1
          RX packets:648599 errors:0 dropped:0 overruns:0 frame:0
          TX packets:984367 errors:0 dropped:0 overruns:0 carrier:0
```

Thanks

**Comment by walterheisenberg** [comment]
I've never used VLAN. Is it only on eth0.x?

**Comment by ryecoaaron** [comment]
Fixed in 3.0.4.
